step-templates/redgate-deploy-from-package.json

```diff
@@ -122,16 +122,25 @@
                                                }
                        },
                        {
-                           "Name":  "SpecificModuleVersion",
+                           "Name":  "DLMAutomationSpecificModuleVersion",
```

Comment:
I understand the desire to want to change this variable name to be consistent, but anyone using this step will be forced to update this value (if they have it set).  Or, they could lose the value they have set already.  Is it absolutely required to change this variable name?

![image](https://user-images.githubusercontent.com/38642825/85149211-a2e28400-b216-11ea-8d92-cbe7bb4ca026.png)